fxa-client/src/errors.rs

```diff
@@ -79,11 +79,14 @@ pub enum ErrorKind {
     #[fail(display = "No cached token for scope {}", _0)]
     NoCachedToken(String),
 
+    #[fail(display = "No cached refresh token")]
```

Comment:
nit: is "cached" an appropriate description here, or should we say something like "no stored refresh token"